pwa/src/templates/templateParts/searchComponent/SearchComponentTemplate.tsx

```diff
@@ -59,12 +60,15 @@ export const SearchComponentTemplate: React.FC<SearchComponentTemplateProps> = (
         </FormFieldInput>
       </FormField>
       <div className={styles.buttons}>
-        <Button type="submit" icon={<SearchIcon />} variant="secondary-action">
+        <Button type="submit" appearance="primary-action-button">
+          <SearchIcon />
           {t("Search")}
         </Button>
-        <Button icon={<ArrowRightIcon />} iconAlign="start" onClick={clearFilters} variant="secondary-action">
+
+        <ButtonLink onClick={clearFilters} appearance="secondary-action-button">
+          <ArrowRightIcon />
           {t("View all components")}
-        </Button>
+        </ButtonLink>
```

Comment:
I think, due to this click having direct side effects, we shouldn't use this `ButtonLink` component here. Please check with @Robbert.

Note: the side effect here is that we're clearing state and only _thereafter_ navigating the user. Instead of _only_ navigating.

Reply:
Probably this approach is already better than it was: previously the link could not be found using a screen reader.

@lencodes Maybe the ideal way method is using the [GatsbyLink state]() property, then the code would look something like this:

```jsx
<GatsbyLink to="/components" state={{
      resultDisplayLayout: filters.resultDisplayLayout,
      dependenciesDisplayLayout: filters.dependenciesDisplayLayout,
      landingDisplayLayout: filters.landingDisplayLayout,
      currentPage: filters.currentPage,
      applicationsCurrentPage: filters.applicationsCurrentPage,
    }} />
```